cob_hardware_config/cob3-2/config/base_controller.yaml

```diff
@@ -37,6 +37,10 @@ twist_controller:
 odometry_controller:
   type: cob_omni_drive_controller/OdometryController
   publish_rate: 50
+  frame_id: "/odom_combined"
```

Comment:
As this PR has "backwards compatibility" in its name, I'm simply making sure that the values are set to the exact same value as before - i.e. including `/`.
Getting rid of the leading slash should happen in a new PR...